services/squareone/templates/configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "squareone.fullname" . }}
+  labels:
+    {{- include "squareone.labels" . | nindent 4 }}
+data:
+  squareone.config.yaml: |
+    siteName: {{ .Values.config.siteName | quote }}
+    baseUrl: {{ .Values.global.baseUrl | quote }}
+    siteDescription: |
+      {{ .Values.config.siteDescription }}
+    {{- if .Values.config.semaphoreUrl }}
+    semaphoreUrl: |
+      {{ .Values.config.semaphoreUrl }}
```

Comment:
If the description contains a newline without the right indentation, I think this may result in invalid YAML. I would tend to just use `| quote` even though this may be a long string and let Helm deal with any special characters, which also has the benefit of making all these settings uniform.